src/pages/layer-2.js

```diff
@@ -0,0 +1,629 @@
+              <Link to="/developers/docs/scaling/zk-rollups/">
+                More on zk-rollups
+              </Link>
+            </p>
+          </Flex50>
+        </TwoColumnContent>
+        <InfoBanner isWarning={true}>
+          <h2>DYOR! Risks of layer 2</h2>
+          <p>
+            Because layer 2 chains inherit security from Ethereum, in an ideal
+            world, they are as safe as L1 Ethereum. However, many of the
+            projects are still young and somewhat experimental. After years of
+            R&D, many of the L2 technologies that will scale Ethereum went live
+            in 2021. This is not to say these L2s are not secure, only that no
+            layer 2 is as battle tested as Ethereum Mainnet. Always do your own
+            research and decide if you're comfortable with any risks involved.
+          </p>
+          <p>
+            For more information on the technology, risks and trust assumptions
+            of layer 2s, we recommend checking out L2BEAT, which provides a
+            comprehensive risk assessment framework of each project.
+          </p>
+          <p>
+            <Link to="https://l2beat.com">L2BEAT</Link>
+          </p>
+        </InfoBanner>
+      </Content>
+
+      <Content>
+        <h2>A note on alt L1s, sidechains, and validiums</h2>
+        <p>
+          Many alternative layer 1s have higher throughput and lower transaction
+          fees than Ethereum. These alt L1s have had to sacrifice on security or
+          decentralization in order to achieve higher transactions per second
+          and lower fees. The Ethereum ecosystem is firmly aligned that layer 2
+          scaling is the only way to solve the scalability trilemma and remain
+          decentralized and secure.
+        </p>
+        <p>
+          Sidechains and validiums are blockchains that allow assets from one
+          blockchain to be bridged over and used on another blockchain.
+          Sidechains and validiums run in parallel with the main chian, and
+          interact with the main chain through bridges, but they do not derive
+          their security or data availability from the main chain. They scale
+          similarly to layer 2s, but have different trust assumptions. They
+          offer lower transaction fees, and higher transaction throughput. More
+          on <Link to="/developers/docs/scaling/sidechains/">sidechains</Link>{" "}
+          and <Link to="/developers/docs/scaling/validium/">validiums</Link>.
+        </p>
+      </Content>
+
+      <Content>
+        <h2>Generalized layer 2s</h2>
+        <p>
+          Generalized layer 2s behave just like Ethereum—but cheaper. Anything
+          that you can do on Ethereum layer 1, you can also do on layer 2. Many
+          dapps have already begun to migrate to these networks, or are skipping
+          deploying to mainnet altogether.
+        </p>
+        <StyledCardGrid>
```

Comment:
What about the non-L2 solutions we wanted to include here?

- Polygon
- Ronin

Reply:
I think moving the note on alt-l1's below lets us add this back in while being clear they arent layer 2's. Will experiment with this a bit.